config/tool_conf.xml.sample

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<toolbox>
+<toolbox monitor="true">
```

Comment:
I have this change if I pull in your changes: `<toolbox monitor="true">`
I don't think it has something do to with `config/*` as all the other config files also get's updated.